src/components/utils/index.js

```diff
@@ -29,6 +29,21 @@ let isIOS = (/iPad|iPhone|iPod/.test(navigator.platform || '') || (navigator.pla
 // Copied from https://github.com/muaz-khan/DetectRTC/blob/master/DetectRTC.js
 export const isDesktop = !(/Android|webOS|BB10|BlackBerry|IEMobile|Opera Mini|Mobile|mobile/i.test(navigator.userAgent || '')) && !isIOS
 
+// To detect hybrid desktop/mobile devices which have a rear facing camera such as the Surface
+export async function isHybrid() {
+  return isDesktop &&
+    navigator.platform === 'Win32' &&
+    await navigator.mediaDevices.getUserMedia(
```

Comment:
I guess the enumerateDevices option didn't work in the end? 😞

Reply:
Unfortunately, no. Enumerate devices won't return relevant information unless the user has already granted permission, which means we would need to call getUserMedia at least once anyway.